paimon-spark/paimon-spark-common/src/test/scala/org/apache/paimon/spark/sql/AnalyzeTableTestBase.scala

```diff
@@ -118,6 +121,12 @@ abstract class AnalyzeTableTestBase extends PaimonSparkTestBase {
         Row(2, 0, 2, "{ }"))
     }
 
+    withSQLConf("spark.paimon.scan.timestamp-millis" -> time1.toString) {
+      checkAnswer(
+        sql("SELECT snapshot_id, schema_id, mergedRecordCount, colstat FROM `T$statistics`"),
```

Comment:
I think you can modify "colstat" to "colStat" in StatisticTable. And then modify other relevant codes.